### ISSUE TITLE
MueLu: Fishy-looking CMake warning about a test

### ISSUE THREAD

**Issue by mhoemmen** [trilinos/Trilinos#1741]
@trilinos/muelu 
The CMake warnings below that "[a]rguments are being passed in but not used" look a bit fishy.  Does that mean that there is some CMake syntax issue that could make the test build or run incorrectly?
```
-- MueLu_HelmholtzFEM2DSerial: NOT added test because TPL_ENABLE_MPI='ON' and COMM='serial'!
CMake Warning at cmake/tribits/core/package_arch/TribitsGeneralMacros.cmake:283 (MESSAGE):
  Arguments are being passed in but not used.  UNPARSED_ARGUMENTS =
  Name;HelmholtzFEM3DSerial
Call Stack (most recent call first):
  cmake/tribits/core/package_arch/TribitsAddTest.cmake:762 (TRIBITS_CHECK_FOR_UNPARSED_ARGUMENTS)
  packages/muelu/test/helmholtz/CMakeLists.txt:87 (TRIBITS_ADD_TEST)

CMake Warning at cmake/tribits/core/package_arch/TribitsGeneralMacros.cmake:283 (MESSAGE):
  Arguments are being passed in but not used.  UNPARSED_ARGUMENTS =
  Name;Helmholtz1DParallel
Call Stack (most recent call first):
  cmake/tribits/core/package_arch/TribitsAddTest.cmake:762 (TRIBITS_CHECK_FOR_UNPARSED_ARGUMENTS)
  packages/muelu/test/helmholtz/CMakeLists.txt:94 (TRIBITS_ADD_TEST)
```

**Comment by aprokop** [comment]
@mhoemmen I'm pretty sure it is case sensitive. I pushed a quick fix in 42d77a9b522657aa2ff6486c04020ce6675a31c3, but please do let us know if the problem persists.
Closing for now.

**Comment by mhoemmen** [comment]
Thanks @aprokop ! :-D  Just wanted to make sure this wasn't a bad thing.  I'll keep an eye out for this.
